src/Extensions/ActivateSudoModeServiceExtension.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace SilverStripe\BehatExtension\Extensions;
+
+use SilverStripe\Control\Session;
+use SilverStripe\Forms\Form;
+use SilverStripe\Core\Extension;
+
+/**
+ * Extension that will make it as if sudo mode is active for the current session
+ * Intended to be used with behat tests so that there is no need to enter a password
+ * when testing forms that require sudo mode
+ *
+ * @extends Extension<Form>
+ */
+class ActivateSudoModeServiceExtension extends Extension
+{
+    public function updateCheck(bool &$active, Session $session)
```

Comment:
Should really be `protected` per best practices but since it's just for tests I'll leave it be